nipype/pipeline/engine/nodes.py

```diff
@@ -577,7 +577,9 @@ def _run_command(self, execute, copyfiles=True):
             self._originputs = deepcopy(self._interface.inputs)
             self._copyfiles_to_wd(execute=execute)
 
-        message = '[Node] Running "{}" ("{}.{}")'
+        message = '[Node] Running "{}" ("{}.{}")'.format(
```

Comment:
None of these variables should contain percent characters. Why not letting the log module do the work?

Reply:
Better to do the substitution first (before cmd is added), since a rouge `%` or `{}` will lead to a logging error